.travis.yml

```diff
@@ -23,8 +42,8 @@ install:
   - sudo update-java-alternatives --set java-8-oracle
   - pip install -r requirements.txt
   - pip install -e .
-  - pip install coveralls
   - pip install pygdal==`gdal-config --version`
+  - pip install codecov
```

Comment:
We uses codecov because it was able to generate aggregated coverage between different environment ran by travis.
I haven't been able to find similar functionality on coveralls.io